src/challenges.js

```diff
@@ -28,16 +52,38 @@ const cacheFunction = cb => {
 
 /* ======================== Recursion Practice ============================ */
 const reverseStr = str => {
```

Comment:
Great solution but we wanted you to use recursion for this one.  Still a good solution.